### PR TITLE
Add mhash and aide

### PR DESCRIPTION
**Pull request by yuwata**
How to test aide:
```
# aide --init
# mv /var/lib/aide/aide.db{.new,}.gz
# aide --update
```

### CODE REVIEW

File: recipes-debian/mhash/mhash_debian.bb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+inherit debian-package
+require recipes-debian/sources/mhash.inc
+
+LICENSE = "GPL-2.0"
```
Comment (hiraku-wfs) [inline recipes-debian/mhash/mhash_debian.bb]: I think it is LGPL.

Comment (yuwata) [reply recipes-debian/mhash/mhash_debian.bb]: Indeed. Thanks. Fixed.

File: recipes-debian/mhash/mhash_debian.bb [hunk 2]
```diff
@@ -0,0 +1,34 @@
+inherit debian-package
+require recipes-debian/sources/mhash.inc
+
+LICENSE = "LGPLv2"
```
Comment (hiraku-wfs) [inline recipes-debian/mhash/mhash_debian.bb]: Some files are licensed under LGPLv2.1, so I think "LGPLv2+" is appropriate.

Comment (yuwata) [reply recipes-debian/mhash/mhash_debian.bb]: Thank you for the detailed check. Updated.

File: recipes-debian/mhash/mhash_debian.bb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+	cp ${WORKDIR}/Makefile.test ${S}/demo/Makefile
+	cp ${WORKDIR}/mhash.c ${S}/demo/
+	cp include/mutils/mhash_config.h ${S}/include/mutils/
+	cp -r lib/.libs ${S}/lib/
+	make -C ${S}/demo CFLAGS="${CFLAGS} -I${S}/include/" LDFLAGS="${LDFLAGS} -L${S}/lib/.libs"
+}
+
+do_install_ptest() {
+	install -m 0755 ${S}/demo/mhash ${D}${PTEST_PATH}
+}
```
Comment (hiraku-wfs) [inline recipes-debian/mhash/mhash_debian.bb]: I found meta-security has a corresponding recipe of libmhash.
https://layers.openembedded.org/layerindex/recipe/101469/

Could you refer to this recipe?

Comment (yuwata) [reply recipes-debian/mhash/mhash_debian.bb]: OK. The header is added. Also, renamed to libmhash from mhash.

**Comment by hiraku-wfs** [comment]
mhash.c and run-ptest are essentially the same as meta-security's ones. Difference is only indent.
Could you use the same file as meta-security's ones for future maintainability?

**Comment by yuwata** [comment]
@hiraku-wfs Thank you for reviewing this so many times. Updated. PTAL.

**Comment by hiraku-wfs** [comment]
Thanks!

**Pull request merged**